temp_git_add.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /home/vibecoder/neonpro
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)